uui/components/pickers/PickerInput.tsx

```diff
@@ -10,11 +10,18 @@ import { DataPickerBody } from './DataPickerBody';
 import { DataPickerRow } from './DataPickerRow';
 import { DataPickerFooter } from './DataPickerFooter';
 import { PickerItem } from './PickerItem';
+import { PickerTogglerTagProps } from './PickerTogglerTag';
 
 const pickerHeight = 300;
 const pickerWidth = 360;
 
-export type PickerInputProps<TItem, TId> = SizeMod & IHasEditMode & PickerInputBaseProps<TItem, TId>;
+export type PickerInputProps<TItem, TId> = SizeMod & IHasEditMode & PickerInputBaseProps<TItem, TId> & {
+    /**
+     * Render callback for picker toggler selection tag
+     * If omitted, default `PickerTogglerTag` component will be rendered
+     */
+    renderTag?: (props: PickerTogglerTagProps<TItem, TId>) => JSX.Element;
```

Comment:
I believe this callback receive IRenderItemProps interface, not PickerTogglerTagProps, yes?

Reply:
fixed